schema/person-statement.json

```diff
@@ -182,6 +167,32 @@
     },
     "replacesStatements": {
       "$ref": "components.json#/definitions/ReplacesStatements"
+    },
+    "pepStatus": {
+      "type": "object",
+      "title": "Politically Exposed Person status",
+      "description": "Information about whether, and how, the person described by this statement is politically exposed. Use this property only if PEP declarations are expected as part of beneficial ownership declarations.",
+      "required": [
+        "hasPepStatus"
+      ],
+      "properties": {
+        "hasPepStatus": {
+          "type": [
+            "boolean",
+            "null"
+          ],
+          "title": "Politically Exposed Person status",
+          "description": "This value is 'true' or 'false' according to whether the person described by this statement has the status of politically exposed person. A null value means a PEP status declaration is expected but missing; the reason for the missing data SHOULD be supplied in the details field."
```

Comment:
If the suggested wording above works, then this description could be:

"This value is 'isPep' or 'isNotPep' according to whether the person described by this statement has the status of politically exposed person. An 'unknown' value means a PEP status declaration is expected but missing; the reason for the missing data SHOULD be supplied in the details field."

Reply:
Agreed with @kindly that this description works.